packages/components/src/components/content-expander/content-expander.jsx

```diff
@@ -20,27 +20,24 @@ const ContentExpander = ({
     onToggle,
     measure,
 }) => {
-    const [is_visible, toggleVisibility] = React.useState(is_expanded);
```

Comment:
Removing this means we'll always need a toggle function declared outside of the component for `is_visible`, the internal state usage offers more flexibility for `ContentExpander` if the component user doesn't wish to pass the callBack prop.